templates/repo/actions/runs_list.tmpl

```diff
@@ -36,15 +36,20 @@
 					<div class="run-list-meta">{{svg "octicon-calendar" 16}}{{DateUtils.TimeSince $run.Updated}}</div>
 					<div class="run-list-meta">{{svg "octicon-stopwatch" 16}}{{$run.Duration}}</div>
 				</div>
-				{{if and ($.AllowDeleteWorkflowRuns) ($run.Status.IsDone)}}
-					<button class="btn interact-bg link-action tw-p-2"
-						data-url="{{$run.Link}}/delete"
-						data-modal-confirm="{{ctx.Locale.Tr "actions.runs.delete.description"}}"
-						data-tooltip-content="{{ctx.Locale.Tr "actions.runs.delete"}}"
-					>
-						{{svg "octicon-trash"}}
-					</button>
-				{{end}}
+				<div class="ui jump dropdown item tw-p-2">
+					{{svg "octicon-kebab-horizontal"}}
+					<div class="menu">
+						<a class="item" href="{{$run.Link}}">{{svg "octicon-play"}}{{ctx.Locale.Tr "view"}}</a>
+						{{if and ($.AllowDeleteWorkflowRuns) ($run.Status.IsDone)}}
+							<button class="item btn interact-bg link-action"
```

Comment:
Why use `<button class="btn interact-bg">`? These styles don't work with menu item.

Reply:
I do not think it is right, we never used `<button class="btn...">` inside any dropdown item.

Abusing the CSS styles would cause various UI problems.

Maybe what you need is `<a class="item link-action" ...>`